src/main.zig

```diff
@@ -63,49 +63,49 @@ pub fn askBool(prompt: []const u8) !bool {
 }
 
 pub fn askSelectOne(prompt: []const u8, comptime options: type) !options {
-    const in = std.io.getStdIn().inStream();
-    const out = std.io.getStdOut().outStream();
+    const in = std.io.getStdIn().reader();
+    const out = std.io.getStdOut().writer();
 
-    _ = try out.write(ansi.Foreground(ansi.Cyan) ++ "? " ++ ansi.Foreground(ansi.White));
-    _ = try out.write(prompt);
-    _ = try out.write(ansi.Foreground(ansi.DarkGray) ++ " (select one)" ++ ansi.Reset() ++ "\n\n");
+    try out.writeAll(ansi.Foreground(ansi.Cyan) ++ "? " ++ ansi.Foreground(ansi.White));
+    try out.writeAll(prompt);
+    try out.writeAll(ansi.Foreground(ansi.DarkGray) ++ " (select one)" ++ ansi.Reset() ++ "\n\n");
 
     comptime var max_size: usize = 0;
     inline for (@typeInfo(options).Enum.fields) |option| {
-        _ = try out.write("  - ");
-        _ = try out.write(option.name);
-        _ = try out.write("\n");
+        try out.writeAll("  - ");
+        try out.writeAll(option.name);
+        try out.writeAll("\n");
 
         if (option.name.len > max_size) max_size = option.name.len;
     }
 
     while (true) {
         var buffer: [max_size + 1]u8 = undefined;
 
-        _ = try out.write(ansi.Foreground(ansi.DarkGray) ++ "\n>" ++ ansi.Reset() ++ " ");
+        try out.writeAll(ansi.Foreground(ansi.DarkGray) ++ "\n>" ++ ansi.Reset() ++ " ");
 
         var result = (in.readUntilDelimiterOrEof(&buffer, '\n') catch {
             try in.skipUntilDelimiterOrEof('\n');
-            _ = try out.write(ansi.Foreground(ansi.Red) ++ "Error: Invalid option, please try again.\n" ++ ansi.Reset());
+            try out.writeAll(ansi.Foreground(ansi.Red) ++ "Error: Invalid option, please try again.\n" ++ ansi.Reset());
             continue;
         }) orelse return error.EndOfStream;
         result = if (std.mem.endsWith(u8, result, "\r")) result[0..(result.len - 1)] else result;
 
         inline for (@typeInfo(options).Enum.fields) |option|
-            if (std.mem.eql(u8, option.name, result))
+            if (std.ascii.eqlIgnoreCase(u8, option.name, result))
```

Comment:
```suggestion
            if (std.ascii.eqlIgnoreCase(option.name, result))
```
As this method is ASCII-specific, it does not take `u8` as a parameter.

Reply:
Sorry, missed this when naively changing it, its not in the library tests either.